web/test/pages/onboarding/onboarding-foreign.test.tsx

```diff
@@ -2,8 +2,13 @@ import { getMergedConfig } from "@/contexts/configContext";
 import { templateEval } from "@/lib/utils/helpers";
 import { generateProfileData, generateUserData } from "@/test/factories";
 import { expectContent } from "@/test/helpers";
+import * as mockRouter from "@/test/mock/mockRouter";
 import { useMockRouter } from "@/test/mock/mockRouter";
```

Comment:
you have two styles of imports for the same thing being imported here

Reply:
we typically avoid `import * as X` imports (there is a single exception to this, which is for when we do module mocking, like of the `api` - happy to talk further on that if need be)

It looks like this was added because you want to use `mockPush` but I think you could just import that directly by adding it to the existing destructuring import line, instead of this new line being necessary

Reply:
Done